bin/centreondb-smooth-backup.sh

```diff
@@ -361,12 +353,12 @@ echo "#####################"
 # We need to stop if need
 ###
 if [ "$PACEMAKER_ON" = "1" ] ; then
-	crm resource unmanage "$PACEMAKER_RSC_MYSQL"
+	pcs resource unmanage "$PACEMAKER_RSC_MYSQL"
 fi
 if [ "$started" -eq 1 ] ; then
 	i=0
 	output_log "Stopping mysqld:" 0 1
-	$INIT_SCRIPT stop
+	systemctl stop mariadb
```

Comment:
there are some edge cases where stopping this foookin daemon is not working well. Use mysqladmin to be sure that everything is stopper

```suggestion
	mysqladmin -u "$DB_ROOT_USER" -p"$DB_ROOT_PASSWORD" shudtdown
```